.github/workflows/release_validate.yml

```diff
@@ -35,16 +35,22 @@ jobs:
 
       - uses: actions/setup-go@v5
         with:
-          go-version: 'stable'
+          go-version: '1.21'
 
       - uses: actions/setup-java@v4
         with:
+          java-version: '11'
           distribution: 'temurin'
-          java-version: '21'
 
       - uses: actions/setup-node@v4
         with:
-          node-version: 'latest'
```

Comment:
Hm, so this intentionally used the latest version of the language, to catch bugs like new node breaking our existing release. But I guess, we need to use the same version as we used for building release for determinism? Can we compute some sort of checksum when preparing the release, and than attach that checksum as another artifact to our release?